pkg/odo/cli/component/ui/ui.go

```diff
@@ -130,7 +130,7 @@ func createComponentNameValidator(context *genericclioptions.Context) survey.Val
 func EnterComponentName(defaultName string, context *genericclioptions.Context) string {
 	var path string
 	prompt := &survey.Input{
-		Message: "How would you wish to name the new component",
+		Message: "How do you wish to name the new component",
```

Comment:
Maybe `What do you wish`?

Reply:
Yeah, I think that reads a little clearer. I pushed up a change to address this. Thanks for your feedback.